task2/server/server.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"strconv"
+	"strings"
+)
+
+func handleAns(conn net.Conn) {
+	defer fmt.Println(conn.RemoteAddr().String(), " closed")
+	defer conn.Close()
+	for {
+		m, err := bufio.NewReader(conn).ReadString('\n')
+		if err != nil {
+			if err != io.EOF {
+				fmt.Println(err)
+			}
+			break
+
+		}
+
+		var resp string
+		m = strings.TrimSuffix(m, string('\n'))
+
+		n, err := strconv.Atoi(m)
+		if err != nil {
+			resp = strings.ToUpper(m)
+		} else {
+			n *= 2
+			resp = strconv.Itoa(n)
+		}
+
+		_, err = conn.Write([]byte(resp + string('\n')))
+		if err != nil {
+			fmt.Println(err)
+			break
+		}
+
+	}
+}
+
+func main() {
+	ln, err := net.Listen("tcp", ":8083")
+	defer ln.Close()
+	if err != nil {
+		log.Fatal("1) ", err)
```

Comment:
What `1) ` does mean ?

Reply:
it helps to find a place, where program felt.

Reply:
This is a debug information, that will be redundant for the client.